geode-apis-compatible-with-redis/src/test/java/org/apache/geode/redis/internal/data/RedisSortedSetTest.java

```diff
@@ -656,7 +656,9 @@ public void redisSortedSetGetSizeInBytes_isAccurateForZpopmax() {
     for (int i = 0; i < numberOfEntries; ++i) {
       sortedSet.zpopmax(mockRegion, mockKey, 1);
       sortedSet.clearDelta();
-      assertThat(sizer.sizeof(sortedSet)).isEqualTo(sortedSet.getSizeInBytes());
+      int actualSize = sizer.sizeof(sortedSet) - sizer.sizeof(ByteArrays.HASH_STRATEGY);
+      int calculatedSize = sortedSet.getSizeInBytes();
+      assertThat(calculatedSize).isEqualTo(actualSize);
```

Comment:
the use of "actual" here is a bit confusing, since in general we do assertThat(actual).isEqualTo(expected). In this case the expected is the "actual" size and actual is "calculated". I think it would be less confusing to use a variable name like "measuredSize". (for this and the above tests)

Reply:
I changed them to expectedSize and actualSize